gateway/api/utils.py

```diff
@@ -182,3 +183,18 @@ def generate_cluster_name(username: str) -> str:
     pattern = re.compile("[^a-zA-Z0-9-.]")
     cluster_name = f"c-{re.sub(pattern,'-',username)}-{str(uuid.uuid4())[:8]}"
     return cluster_name
+
+
+def sanitize_file_path(path: str):
```

Comment:
is there a way we can use the one from the client utils rather than duplicating the code here? (if not, maybe we need a high-level utils package?)

Reply:
I took out one duplicate function in the gateway.  There still one in each client and gateway.  The gateway doesn't import the client.  I want to keep that way.

Reply:
Agree on not importing client in gateway. 
I'll open an issue about utils common across gateway and client. In an ideal world we wouldn't duplicate code, but for just one utility this may be the easy path.

Reply:
If some code needs to be shared usually it is formed into a separate library. It is starting to make sense when we have decent amount of duplicate code. Couple of functions is fine :)